util/find-doc-nits

```diff
@@ -512,14 +509,15 @@ sub checklinks {
     }
 }
 
+# Load the public symbol/macro names
 sub publicize {
     foreach my $name ( parsenum('util/libcrypto.num') ) {
         $public{$name} = 1;
     }
     foreach my $name ( parsenum('util/libssl.num') ) {
         $public{$name} = 1;
     }
-    foreach my $name ( parsenum('util/private.num') ) {
+    foreach my $name ( parsenum('util/other-names.num') ) {
```

Comment:
Uhmm...

Reply:
rats, that's the second time I've forgotten that.  updated commit pushed.